### PR TITLE
Road to 1.0 Release

### PR DESCRIPTION
**Pull request by charlespierce**
> Document the features necessary to release Volta 1.0

[Rendered RFC](https://github.com/charlespierce/rfcs/blob/road_to_10/text/0000-road-to-10.md)

### CODE REVIEW

File: text/0000-road-to-10.md [hunk 1]
```diff
@@ -57,7 +57,11 @@ There are a number of issues that are relatively small on their own, but combine
...
 
-Another feature that could be considered for 1.0 is support for Yarn Plug'n'Play, which installs all of the dependencies from a local cache and doesn't use `node_modules`. However, we haven't yet received requests to support that functionality, and it will require a fair amount of investigation on the implementation. One major issue is that Yarn's feature relies on JS Monkeypatching, which we can't take advantage of in Rust, so we would need to either call Node (which will have performance implications) or duplicate the Yarn logic within Volta (which will take significant development effort).
+A feature that could be considered for 1.0 is support for Yarn Plug'n'Play, which installs all of the dependencies from a local cache and doesn't use `node_modules`. However, we haven't yet received requests to support that functionality, and it will require a fair amount of investigation on the implementation. One major issue is that Yarn's feature relies on JS Monkeypatching, which we can't take advantage of in Rust, so we would need to either call Node (which will have performance implications) or duplicate the Yarn logic within Volta (which will take significant development effort).
```
Comment (chriskrycho) [inline text/0000-road-to-10.md]: Note that while we haven't *yet* hit this, some puzzle pieces *should* 🤞 fall into place in the Ember community (many of our early adopters) over the next few months that unlock Yarn P'n'P there, making adoption for some of our users much likelier.

Comment (charlespierce) [reply text/0000-road-to-10.md]: Good to know :+1: We'll definitely need to keep an eye on that and bump it up in priority if it becomes a requested feature.

**Comment by dherman** [comment]
I suspect we can remove PnP support from the list of 1.0 blockers. So far it seems like PnP has enough adoption issues that it's not being widely used yet and may evolve.

**Comment by rwjblue** [comment]
> I suspect we can remove PnP support from the list of 1.0 blockers.

Ya, seems fine to me. It also seems like something we can do within the 1.x timeframe without any backwards incompatible changes.

File: text/0000-road-to-10.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+## Apple Silicon Support
+
+The new Apple processors are available in laptops and showing lots of promise. Though we should support native builds on those processors, currently Volta works correctly under Rosetta 2, so this isn't a blocker for a 1.0 release. When we do add support, we also need to take into account that we may need to fall back to downloading x64 versions of Node on Apple devices, since there may not be native builds available for older versions of Node.
+
+## pnpm Support
+
+Currently, Volta only supports npm and Yarn for package managers, as they are the two most popular. [pnpm](https://pnpm.js.org/) is a relative newcomer to the space, however it is growing in adoption and we have received a few requests to support it in Volta. However, there are some technical hurdles (most notably pnpm's clobbering of `NODE_PATH` when running executables) that make it difficult to implement at the moment. While pnpm support is definitely on the near-term roadmap (likely one of the first features to be tackled after 1.0), the volume of users requesting `pnpm` is low enough that it isn't necessary as part of the 1.0 feature set.
```
Comment (dherman) [inline text/0000-road-to-10.md:57]: A couple slight verbiage suggestions to really emphasize that we care about supporting pnpm:

> However, there are some technical hurdles

I'd change this to "There are a few remaining technical hurdles (most notably etc etc) blocking the implementation at the moment" so it doesn't sound like we have doubts about whether we _can_ do it, just a few steps left to go, with a sense of inevitability that we'll get there.

> the volume of users requesting `pnpm` is low enough that it isn't necessary as part of the 1.0 feature set

This is a little unnecessarily dismissive. Maybe something like "While pnpm support is definitely on the near-term roadmap (likely one of etc etc), the core of Volta has stabilized to the point that we don't need to block shipping for the large number of users of other package managers."

**Pull request merged**